docs/rules/init-declarations.md

```diff
@@ -111,10 +115,14 @@ function foo() {
 }
 ```
 
-With `"ignoreForLoopInit"` enabled, the following pattern is not considered a problem.
+The `"never"` option ignores `const` variable initializations.
+
+### ignoreForLoopInit
+
+Examples of **correct** code for the `"never", {"ignoreForLoopInit": true}` options:
 
 ```js
-/*eslint init-declarations: [2, "never", { "ignoreForLoopInit": true }]*/
+/*eslint init-declarations: [2, "never", {"ignoreForLoopInit": true}]*/
```

Comment:
I'm not sure how I feel about this change.  @pedrottimark you've been looking at these docs a lot lately, what would you say we mostly use, spaces inside object literals, or no spaces?  I'd like us to be consistent, and this change would cause inconsistency at least within this file (see https://github.com/eslint/eslint/pull/5375/files#diff-4e01e127f3ace0684323c2943aef2d5cR58).  Personally I prefer the spaces for readability, and our own `eslint-config-eslint` enforces `block-spacing: 2`.